spec/lib/twingly/http_spec.rb

```diff
@@ -605,4 +605,14 @@ class CustomError < StandardError; end
       end
     end
   end
+
+  describe "#initialize" do
```

Comment:
there are already some other tests that concerns logging, perhaps it could be moved there somehow. Something regarding the default logger. Also the specs have a `NullLogger` class, I assume that is not needed anymore

Reply:
Hm, I can't find a natural place for the test anywhere else. 

I don't think the `NullLogger` class was needed prior to this change either, but it could indeed be changed into `Logger.new("/dev/null")`.

Reply:
NullLogger was removed in 79ddb53